### PR TITLE
increases max zoom levels for raster uploads

### PR DESCRIPTION
**Pull request by isiyu**
refs https://github.com/mapbox/unpacker/issues/852

@dnomadb  does this need more [`SPATIAL RESOLUTIONS` tests](https://github.com/mapbox/mapnik-omnivore/commit/da07395feaad80f4816b68f7b6e4b33f8ea8fbfa#diff-02c0008a8c3721e69b6d0fa9e044129cR401) for the new zoom levels?

### CODE REVIEW

**Comment by dnomadb** [comment]
> does this need more SPATIAL RESOLUTIONS tests for the new zoom levels?

@isiyu y.

File: test/raster.test.js [hunk 1]
```diff
@@ -410,9 +410,9 @@ tape('[SPATIAL RESOLUTIONS] Get spatial resolutions / valid spatial resolutions'
   assert.end();
 });
 
-tape('[SPATIAL RESOLUTIONS] Get spatial resolutions / valid spatial resolutions with weight 0.5', function(assert) {
+tape('[SPATIAL RESOLUTIONS] Get spatial resolutions / valid spatial resolutions with weight 0.25', function(assert) {
```
Comment (dnomadb) [inline test/raster.test.js]: Curious @isiyu  - why did this test change?

File: test/raster.test.js [hunk 2]
```diff
@@ -400,7 +400,7 @@ tape('[TIFF] JPEG-compressed', function(assert) {
 
 tape('[SPATIAL RESOLUTIONS] Get spatial resolutions / valid spatial resolutions', function(assert) {
   var spatialResolutions = utils.getSpatialResolutions();
-  var expectedResolutions = JSON.parse('[156542.96875,78271.484375,39135.7421875,19567.87109375,9783.935546875,4891.9677734375,2445.98388671875,1222.991943359375,611.4959716796875,305.74798583984375,152.87399291992188,76.43699645996094,38.21849822998047,19.109249114990234,9.554624557495117,4.777312278747559,2.3886561393737793,1.1943280696868896,0.5971640348434448,0.2985820174217224]');
+  var expectedResolutions = JSON.parse('[ 156542.96875, 78271.484375, 39135.7421875, 19567.87109375, 9783.935546875, 4891.9677734375, 2445.98388671875, 1222.991943359375, 611.4959716796875, 305.74798583984375, 152.87399291992188, 76.43699645996094, 38.21849822998047, 19.109249114990234, 9.554624557495117, 4.777312278747559, 2.3886561393737793, 1.1943280696868896, 0.5971640348434448, 0.2985820174217224, 0.1492910087108612, 0.0746455043554306 ]');
```
Comment (dnomadb) [inline test/raster.test.js]: @isiyu because this is stringified json, you can keep no spaces.

File: test/raster.test.js (hunk 1 above)
Comment (isiyu) [inline test/raster.test.js]: I changed the description - it said it was with weight 0.5  but the call below on line 418 was using weight 0.25

Comment (dnomadb) [inline test/raster.test.js]: 💭 ... 👍

File: lib/utils.js [hunk 3]
```diff
@@ -87,7 +87,7 @@ module.exports.getUnitType = function(srs) {
 
 module.exports.getSpatialResolutions = function() {
   var circumference = 40075000;
-  var zoomLevels = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19];
+  var zoomLevels = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20];
```
Comment (GretaCB) [inline lib/utils.js]: Why not bump to z21?

Comment (dnomadb) [inline lib/utils.js]: @GretaCB @isiyu maxzoom is `zoomLevels.length` (https://github.com/mapbox/mapnik-omnivore/blob/moremaxzoom/lib/raster.js#L179), which means that this _is_ 21. That said, previous `maxzoom` was `20` (not 19, as I think we've assumed / verified on master and on branch setup from modularization https://github.com/mapbox/mapnik-omnivore/commit/a0c4cdaec2181dad494c82ca4c5f91ebfb44314b), so I think bumping it to `21` (maxzoom of `22` - 2 more zooms) would be 👍

**Pull request merged**